modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/networkapi/NetworkApiService.scala

```diff
@@ -0,0 +1,68 @@
+package org.tessellation.rosetta.domain.networkapi
+
+import cats.data.EitherT
+import cats.effect.Async
+import cats.syntax.functor._
+
+import org.tessellation.BuildInfo
+import org.tessellation.rosetta.domain.NetworkIdentifier
+import org.tessellation.rosetta.domain.error.{LatestSnapshotNotFound, NetworkApiError}
+import org.tessellation.rosetta.domain.networkapi.model.options._
+import org.tessellation.rosetta.domain.networkapi.model.status._
+import org.tessellation.schema.node.NodeState
+import org.tessellation.schema.timestamp.SnapshotTimestamp
+import org.tessellation.schema.{GlobalIncrementalSnapshot, SnapshotOrdinal}
+import org.tessellation.sdk.config.AppEnvironment
+import org.tessellation.security.Hashed
+import org.tessellation.security.hash.Hash
+import org.tessellation.security.hex.Hex
+
+trait NetworkApiService[F[_]] {
+  def list(appEnv: AppEnvironment): EitherT[F, NetworkApiError, List[NetworkIdentifier]]
+  def options: EitherT[F, NetworkApiError, NetworkApiOptions]
+  def status: EitherT[F, NetworkApiError, NetworkStatusResponse]
+}
+
+object NetworkApiService {
+  private val rosettaVersion = "1.4.14" // TODO get from a config?
+  private val version = Version(rosettaVersion = rosettaVersion, nodeVersion = BuildInfo.version)
+
+  type LastSnapshotInfo = (Hashed[GlobalIncrementalSnapshot], SnapshotTimestamp)
+
+  def make[F[_]: Async](
+    lastSnapshot: F[Option[LastSnapshotInfo]],
+    genesisOrdinalAndHash: F[(SnapshotOrdinal, Hash)],
+    nodeState: F[NodeState]
+  ): NetworkApiService[F] = new NetworkApiService[F] {
+    def list(appEnv: AppEnvironment): EitherT[F, NetworkApiError, List[NetworkIdentifier]] =
+      EitherT.pure(NetworkIdentifier.fromAppEnvironment(appEnv).toList)
+
+    def options: EitherT[F, NetworkApiError, NetworkApiOptions] =
+      EitherT.pure(NetworkApiOptions(version, Allow.default))
```

Comment:
if these are pure fns, why wrapping into `EitherT`?
it can be simply:
```scala
def list(appEnv: AppEnvironment): List[NetworkIdentifier] = ...
def options: NetworkApiOptions = ...
```